unleash-yggdrasil/src/state.rs

```diff
@@ -29,6 +31,7 @@ impl EnrichedContext {
             properties: context.properties,
             external_results,
             toggle_name,
+            runtime_hostname: None,
```

Comment:
The magic - generally if we're using the old paths, we don't need this property

This could benefit from some refactor in a future PR but this is big enough already